### DISCUSSION TITLE
请问这个论文模板可以用来排版硕士开题报告吗？

### DISCUSSION THREAD

**Discussion by Sergey3557** [tuna/thuthesis#982] [Q&A]
如果可以的话，请问应该如何设置调整到开题报告的模板格式呢？

**Comment by Youngzipper** [comment]
硕士开题报告因为比较短建议用word，并且没有固定的格式要求（我们系是这样的

**Comment by Harry-Chen** [comment]
我不清楚硕士的开题有什么特别的要求，现有的开题模板（proposal）为什么不能用？
